next/src/app/auth/page.tsx

```diff
@@ -91,8 +91,8 @@ export default function AuthHome() {
   }, [setCompletionCards]);
 
   return (
-    <div className="flex items-center justify-center h-full flex-col gap-10 my-20">
-      <div className="flex justify-center flex-col items-center gap-10 w-2/3 border-4 rounded-md border-black p-10">
+    <div className="flex items-center justify-center h-full flex-col gap-10">
+      <div className="flex justify-center flex-col items-center gap-10 w-3/4 border-4 rounded-md border-black p-10 mt-20">
```

Comment:
Looks good! Try border-[2.5px] and see how you like it (also in the need more help section below)

Other than that, everything looks great!